docs/pages/enterprise/sso.mdx

```diff
@@ -251,13 +457,22 @@ values to match your identity provider:
 
 At this time, the `spec.provider` field should not be set for any other identity providers.
 
-## Working with External Email Identity
+</ScopedBlock>
+
+## Working with an external email identity
 
 Along with sending groups, an SSO provider will also provide a user's email address.
 In many organizations, the username that a person uses to log in to a system is the
-same as the first part of their email address, the "local" part. For example, `dave.smith@acme.com` might log in with the username `dave.smith`. Teleport provides an easy way to extract the first part of an email address so it can be used as a username. This is the `{{email.local}}` function.
+same as the first part of their email address, the "local" part. 
 
-If the email claim from the identity provider (which can be accessed via `{{external.email}}`) is sent and contains an email address, you can extract the "local" part of the email address before the @ sign like this: `{{email.local(external.email)}}`
+For example, `dave.smith@example.com` might log in with the username `dave.smith`.
+Teleport provides an easy way to extract the first part of an email address so
+it can be used as a username. This is the `{{email.local}}` function.
+
+If the email claim from the identity provider (which can be accessed via
```

Comment:
Informational only, but we'll be adding support this quarter to specify an alternate claim for reading the username from, which may render this note unnecessary.